source/_integrations/thread.markdown

```diff
@@ -16,12 +16,30 @@ ha_zeroconf: true
 
 The Thread integration helps you track the different Thread networks in your home and helps you manage their credentials. It is currently a work in progress.
 
-Thread is a low-power mesh networking standard which allows users to connect their devices within a home network. It uses the same RF technology as Zigbee, but provides IP connectivity similar to Wi-Fi. Unlike Zigbee, Thread does not allow to control devices directly: It is just a communication protocol. A higher level protocol such as Matter or HomeKit is required to control Thread enabled devices. Thread allows devices to communicate with each other without the need for a central gateway or router. This makes it ideal for home automation, where a large number of devices may need to be connected.
+##### A networking standard and communication protocol - not a device control standard
```

Comment:
Does the header order/level feel out of order? The page title is header 1; this is below it, but is header level 5?